### ISSUE TITLE
[Suggestion] Whitelist Inventory/Armor by dimension

### ISSUE THREAD

**Issue by katubug** [quat1024/BrokenWings#3]
Right now, you can add multiple items to the whitelist in order to allow flight in blacklisted dimensions. But what I'd really like is the ability to have one item or armor piece per dimension. For example, in order to fly in the Betweenlands, you need to carry/wear the Ring of Recruitment (drop from the final boss). But in the Twilight Forest, you need to have the Lamp of Cinders or something.

Thanks for the helpful mod, and for your time!

**Comment by quat1024** [comment]
This was tricky but I think it works, try the new version

**Comment by quat1024** [comment]
published as 2.1.0 on curse (beta file)

**Comment by katubug** [comment]
Awesome, I'll try it out! Thank you!

**Comment by katubug** [comment]
Just wanted to follow up and say that everything seems to be working perfectly. I ended up not using the item that required metadata (sorry, haha), because I just made my own items with contenttweaker. But I set one item for betweenlands, one for twilight forest, and then one for all dimensions which is crafted with the previous two.

I was testing with only flight from the Wings mod, and everything functioned exactly as expected! I'm super pleased, thank you so much for your work!
